README.md

```diff
@@ -14,19 +14,40 @@ The mission of the Financial Desktop Connectivity and Collaboration Consortium (
 ## About this Repository
 FDC3 is a standard.  This repository houses the documentation and specifications for the ratified FDC3 standards. 
 
+## Contribute to the FDC3 Standard
```

Comment:
@agitana I think how to use the standard is more important than how to contribute, so the contribution section should therefore come after the usage section.

Reply:
thanks @rikoe I've rearranged that section